Dockerfile

```diff
@@ -7,13 +8,15 @@ RUN dotnet restore
 
 # Copy everything else and build
 COPY ./src/* ./
-COPY ./src/queries/get_daily_or_monthly_costs.json ./queries/get_daily_or_monthly_costs.json
+COPY ./src/queries/get_daily_or_monthly_costs.json ./custom_queries/get_daily_or_monthly_costs.json
+COPY ./src/queries/azure_billing_by_resource_group.json ./custom_queries/azure_billing_by_resource_group.json
```

Comment:
I think, you should not copy these files into Docker image

You should use ConfigMap/volumeMount at helm chart to bring these files to the app